service/src/main/java/org/eclipse/keti/acs/policy/evaluation/cache/PolicyEvaluationRequestCacheKey.java

```diff
@@ -18,37 +18,39 @@
 
 package org.eclipse.keti.acs.policy.evaluation.cache;
 
+import java.util.Collections;
 import java.util.LinkedHashSet;
 
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.commons.lang.builder.HashCodeBuilder;
-
 import org.eclipse.keti.acs.rest.PolicyEvaluationRequestV1;
 
 public class PolicyEvaluationRequestCacheKey {
+    public static final String ANY_POLICY_SET_KEY = "any-policy-set";
+    public static final LinkedHashSet<String> EVALUATION_ORDER_SINGLE_POLICY_SET = new LinkedHashSet<>(
```

Comment:
`ANY_POLICY_SET_KEY` - i would use the same name as the key here 'cos it represents it better and always has the one key in it anyway.